src/components/Profile/MainProfile.js

```diff
@@ -82,27 +82,25 @@ function MainProfile({ profile }) {
           >
             <div>
               {email && (
-                <div
+                <p
                   css={`
                     overflow: hidden;
                     white-space: nowrap;
                     text-overflow: ellipsis;
                   `}
                 >
                   {email}
-                </div>
+                </p>
               )}
               {website && (
-                <div
+                <p
                   css={`
                     color: ${theme.contentSecondary};
-                    overflow: hidden;
-                    white-space: nowrap;
-                    text-overflow: ellipsis;
+                    word-break: break-word;
```

Comment:
shouldn't we keep the `...` on overflow?

Reply:
In this case we are breaking the word, so we can be sure that it won't overflow.

e.g
**Before**
![Screen Shot 2020-10-21 at 11 51 00 PM](https://user-images.githubusercontent.com/22663232/96818780-8474db00-13f8-11eb-9a81-192d6bbc4028.png)

**After**
![Screen Shot 2020-10-21 at 11 52 20 PM](https://user-images.githubusercontent.com/22663232/96818790-8c347f80-13f8-11eb-8a34-b49d10697a24.png)